schema/sale_artworks.js

```diff
@@ -0,0 +1,22 @@
+import gravity from '../lib/loaders/gravity';
+import SaleArtwork from './sale_artwork';
+import {
+  GraphQLString,
+  GraphQLList,
+} from 'graphql';
+
+const SaleArtworks = {
+  type: new GraphQLList(SaleArtwork.type),
+  description: 'A list of Sale Artworks',
+  args: {
+    ids: {
+      type: new GraphQLList(GraphQLString),
+      description: 'A list of sale artwork ids',
+    },
```

Comment:
![](https://s-media-cache-ak0.pinimg.com/originals/4d/32/f1/4d32f142871c29466f303c2c80f24ed4.gif)

I'm definitely cool with the composed query you show